.github/workflows/ci.yml

```diff
@@ -0,0 +1,131 @@
+name: Continuity
+
+on:
+  push:
+    branches: [ master ]
```

Comment:
suggest all branches for now or add lint to travis?

Reply:
there are no other branches today; if we ever added one we would have to add this action on that branch and configure it, similar to how we handled `release/1.3` adding actions in the main repo.